tests/providers/google/cloud/operators/test_dataproc.py

```diff
@@ -246,9 +247,11 @@ def test_execute(self, mock_hook):
             timeout=TIMEOUT,
             metadata=METADATA,
         )
+        to_dict_mock.assert_called_once_with(mock_hook().create_cluster().result())
 
+    @mock.patch(DATAPROC_PATH.format("Cluster.to_dict"))
     @mock.patch(DATAPROC_PATH.format("DataprocHook"))
-    def test_execute_if_cluster_exists(self, mock_hook):
+    def test_execute_if_cluster_exists(self, mock_hook, _):
```

Comment:
```suggestion
    def test_execute_if_cluster_exists(self, mock_hook, to_dict_mock):
```
And `to_dict_mock.assert_called_once_with(mock_hook().get_cluster())` at the end of this test. WDYT?

Reply:
Don't we test it in previous test?